content/blog/2022-05-04-out-in-the-open-april-2022.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Informationsfreiheit, kleine Schritte und doch noch ein langer Weg  – Out in the Open April 2022"
+author: "Klara Juhl - OK Lab Osnabrück"
+date: '2022-05-04T09:45:00+01:00'
+excerpt: Monatlich erscheinender Überblick über Entwicklungen in der Open Data und Civic Tech Szene
+topic: community
+images:
+- imgname: out-in-the-open-2022-04-header.jpg
+  attribution: Foto von Marek Slusarczyk](https://commons.wikimedia.org/wiki/User:Tupungato) auf [Wiki Commons](https://commons.wikimedia.org/wiki/File:17_Open_sign_-_free_stock_photo.jpg)
+og_image: out-in-the-open-2022-04-header.jpg
+type: blog
+---
+**Wie steht es um die Informationsfreiheit?**
```

Comment:
Why not use proper markdown headings?  https://raw.githubusercontent.com/okfde/codefor.de/main/content/blog/2022-01-31-out-in-the-open-januar-2022.md did that, the one from February used `**…**` (which is for bold text instead of headings).

```markdown
# H1
## H2
### H3
#### H4
##### H5
###### H6
```
↓

# H1
## H2
### H3
#### H4
##### H5
###### H6

Reply:
true, changed it. Image size is now also reduced, although I noticed that there have been images used for former OutInTheOpen blogs that were up 1.5 MB large.